src/runtime/lpython_intrinsic_numpy.py

```diff
@@ -165,3 +165,63 @@ def _lfortran_sacos(x: f32) -> f32:
 @vectorize
 def arccos(x: f32) -> f32:
     return _lfortran_sacos(x)
+
+########## arctan ##########
+
+@ccall
+def _lfortran_datan(x: f64) -> f64:
+    pass
+
+@overload
+@vectorize
+def arctan(x: f64) -> f64:
+    return _lfortran_datan(x)
+
+@ccall
+def _lfortran_satan(x: f32) -> f32:
+    pass
+
+@overload
+@vectorize
+def arctan(x: f32) -> f32:
+    return _lfortran_satan(x)
+
+########## degrees ##########
+
+@ccall
+def _lfortran_ddegrees(x: f64) -> f64:
+    pass
+
+@overload
+@vectorize
+def degrees(x: f64) -> f64:
+    return _lfortran_ddegrees(x)
```

Comment:
Just implement the formula here. There isn't a need to call C-runtime just for multiplication-division operation.